code/controllers/subsystems/statpanel.dm

```diff
@@ -0,0 +1,117 @@
+SUBSYSTEM_DEF(statpanels)
+	name = "Stat Panels"
+	wait = 4
+	init_order = INIT_ORDER_STATPANELS
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+	var/list/currentrun = list()
+	var/encoded_global_data
+	var/mc_data_encoded
+	var/list/cached_images = list()
+
+/datum/controller/subsystem/statpanels/fire(resumed = FALSE)
+	if (!resumed)
+		var/datum/map_config/cached = SSmapping.next_map_config
```

Comment:
Map config related things can be found here https://github.com/tgstation/tgstation/blob/17e6dc38d7da5e16a62c28fbb5313f66a295ade6/code/datums/map_config.dm

(Errors on line 13, 14, 16, 17, 18)